test/e2e/gameserver_test.go

```diff
@@ -1525,69 +1525,69 @@ func TestLists(t *testing.T) {
 	}{
 		"GetListCapacity": {
 			msg:  "GET_LIST_CAPACITY games",
-			want: "50",
+			want: "CAPACITY: 50\n",
 		},
 		"SetListCapacity": {
 			msg:          "SET_LIST_CAPACITY foo 1000",
-			want:         "true",
+			want:         "SUCCESS: true\n",
 			listName:     "foo",
-			wantCapacity: "1000",
+			wantCapacity: "CAPACITY: 1000\n",
 		},
 		"SetListCapacity past 1000": {
 			msg:          "SET_LIST_CAPACITY games 1001",
 			want:         "ERROR: false\n",
 			listName:     "games",
-			wantCapacity: "50",
+			wantCapacity: "CAPACITY: 50\n",
 		},
 		"SetListCapacity negative": {
 			msg:          "SET_LIST_CAPACITY games -1",
 			want:         "ERROR: false\n",
 			listName:     "games",
-			wantCapacity: "50",
+			wantCapacity: "CAPACITY: 50\n",
 		},
 		"ListContains": {
 			msg:  "LIST_CONTAINS games game2",
-			want: "true",
+			want: "FOUND: true\n",
 		},
 		"ListContains false": {
 			msg:  "LIST_CONTAINS games game0",
-			want: "false",
+			want: "ERROR: false\n",
```

Comment:
This test will need to change with the update to the response.

Reply:
Sorry, I thought the response for `ListContains false` should be `ERROR: false`. I have now updated it as per your suggestion. The response for the `ListContains` is `FOUND: true` and for `ListContains false` it is `FOUND: false`.

Reply:
👍🏻 no worries - had a feeling my suggestion wasn't 100% clear. I feel like this will break the e2e test, since a new image.

This all LGTM! We just need it to pass! Probably best bet is to increment the tag again everywhere and push a new image.

Reply:
Will increment the tag now

Reply:
I've pushed the new image to the agones-images project and updated the necessary files with the new tag.

Reply:
🤞🏻

Reply:
Success! The build has finished without any issues.